### ISSUE TITLE
Could not connect message, even though it works

### ISSUE THREAD

**Issue by mariusa** [chatwoot/chatwoot-mobile-app#599]
Confirm you’re filing a new issue. [Search issues in Github](https://github.com/chatwoot/chatwoot-mobile-app/issues) and ensure that the issue is not already exist.

Checked

**Describe the bug**

Sending a message on Android app shows "Could not connect" message, even though it works, the message is delivered

**To Reproduce**

See notification of a new message received
Open Chatwoot mobile app
Reply
See message

**Screenshots**

![Screenshot_2022-11-25-11-10-19-117_com chatwoot app](https://user-images.githubusercontent.com/1283842/203944199-dd0b2149-868f-4eb9-a1b4-d305827659a0.jpg)

**Observed behavior**
**Environment**

  - Describe whether you are using Chatwoot Cloud - yes
  - Device Name:  Xiaomi Mi 11 5G
  - OS Version: Android 11
  - Chatwoot App Version: latest from app store
    - Chatwoot Server Version: hosted by you

**Comment by muhsin-k** [comment]
Are you using v1.6.18?

**Comment by mariusa** [comment]
Yes, right. It auto-updates from Play store

**Comment by muhsin-k** [comment]
@mariusa Thank you for reporting the issue. We will fix this issue in v1.7.0

**Comment by mariusa** [comment]
Thank you, Muhsin!

**Comment by muhsin-k** [comment]
Fixed via https://github.com/chatwoot/chatwoot-mobile-app/pull/603.
@mariusa Please join our beta testing (https://play.google.com/apps/testing/com.chatwoot.app) and install the version `1.7.7`. We will release it once the testing is complete.
